src/osd/OSD.cc

```diff
@@ -4070,6 +4070,10 @@ void OSD::handle_osd_ping(MOSDPing *m)
               failure_pending[from].second);
             failure_pending.erase(from);
           }
+
```

Comment:
i think 076cb788348e6674345f262ef6b36ae2a1fe402e supersedes this commit. what do you think?